tests/files/validate-ci.yml

```diff
@@ -17,10 +17,6 @@
 - name: Validate zuul status
   hosts: zuul
   tasks:
-    - name: zuul webapp is running
-      uri:
-        url: http://localhost:8001/status.json
```

Comment:
validate-ci.yml doesnt ever run against a v3, so we should be able to avoid dropping this